test/http.spec.ts

```diff
@@ -75,6 +83,38 @@ describe("http", () => {
       });
   });
 
+  it("delete", () => {
+    const testHeaders = {
+      "test-header-key": "Test-Header-Value",
+    };
+
+    return deleteRequest(`${TEST_URL}/delete`, testHeaders).then((res: any) => {
+      equal(res.method, "DELETE");
+      equal(res.path, "/delete");
+      equal(res.headers["test-header-key"], testHeaders["test-header-key"]);
+      equal(res.headers["user-agent"], `${pkg.name}/${pkg.version}`);
+    });
+  });
+
+  it("postBinary", () => {
+    const testHeaders = {
+      "test-header-key": "Test-Header-Value",
+    };
+
+    const filepath = join(__dirname, "/helpers/LINE_Icon.png");
+    const buffer = readFileSync(filepath);
+    return postBinary(`${TEST_URL}/post`, testHeaders, buffer)
+      .then(s => getStreamData(s))
+      .then((data: string) => {
+        const res = JSON.parse(data);
+        equal(res.method, "POST");
+        equal(res.path, "/post");
+        equal(res.headers["test-header-key"], testHeaders["test-header-key"]);
+        equal(res.headers["user-agent"], `${pkg.name}/${pkg.version}`);
+        equal(res.headers["content-type"], "image/png");
```

Comment:
Can you add an assert to check if the submitted file is the same as origin? A `multipart/form-data` parser need to be added to test server.

Reply:
as you know, the api requires `image/png` or `image/jpeg` for `Content-Type`.
https://developers.line.me/ja/docs/messaging-api/reference/#anchor-4f3b10c3de51f6728025dc81cc8f1efc63fd6235
so `postBinary` doesn't uses `multipart/form-data`.
I considering a way such as pipe request body to writable file stream.
Do you have better idea to achieve it?

Reply:
Ah, you're right. We can make this into a separate work. I will investigate on this after this PR is merged, so please do not care of this comment 😅